atomic_reactor/plugins/prepub_squash.py

```diff
@@ -70,13 +72,23 @@ def __init__(self, tasker, workflow, tag=None, from_base=True, from_layer=None,
         self.tag = tag or str(self.workflow.builder.image)
         self.from_layer = from_layer
         if from_base and from_layer is None:
-            try:
-                base_image_id = self.workflow.base_image_inspect['Id']
-            except KeyError:
-                self.log.error("Missing Id in inspection: '%s'", self.workflow.base_image_inspect)
-                raise
-            self.log.info("will squash from base-image: '%s'", base_image_id)
-            self.from_layer = base_image_id
+            if base_image_is_scratch(self.workflow.builder.base_image.to_str()):
+                self.log.debug("build based on scratch, will not include base"
+                               "image in squash")
+                try:
+                    self.from_layer = find_bottom_layer_id(tasker, self.image)
```

Comment:
I think you can just specify `from_layer=None` to squash all the layers: https://github.com/goldmann/docker-squash/blob/05cd17fe639183e2986fa89b3ddb62bcf903cf2b/README.rst